src/ring_theory/algebra_operations.lean

```diff
@@ -222,6 +222,23 @@ begin
     exact subset_span ⟨a, set.mem_singleton a, m, hm, rfl⟩ }
 end
 
+section quotient
+/-- The elements of `I / J` are the `x` such that `x J ⊆ I`.
+
+This is the general form of the ideal quotient, traditionally written $I : J$.
+-/
+instance : has_div (submodule R A) :=
+⟨ λ I J, {
+carrier := { x | ∀ y ∈ J, x * y ∈ I },
```

Comment:
If you want, you can use `pointwise_mul` for this definition. Then you would be able to write `x \bullet J \sub I`. But that probably has slightly worse definitional properties.

Reply:
It is less convenient to work with, so I decided not to change the definition. Instead I added a `mem_div_iff_smul_subset` lemma that converts between the two.